Sources/Commands/SwiftTool.swift

```diff
@@ -580,8 +584,83 @@ public class SwiftTool<Options: ToolOptions> {
         }
     }
 
-    /// Build a subset of products and targets using llbuild.
-    func build(plan: BuildPlan, subset: BuildSubset) throws {
+    func build(subset: BuildSubset) throws {
+        let buildParameters = try self.buildParameters()
+
+        let haveBuildManifestAndDescription =
+            localFileSystem.exists(buildParameters.llbuildManifest) &&
+            localFileSystem.exists(buildParameters.buildDescriptionPath)
+
+        // Run the build with the existing build description if we're asked to by-pass build planning.
+        if options.skipBuildPlanning && haveBuildManifestAndDescription {
+            let buildDescription = try BuildDescription.load(from: buildParameters.buildDescriptionPath)
+            return try build(buildDescription: buildDescription, subset: subset)
+        }
+
+        // Perform steps for build manifest caching if we can enabled it.
+        let hasEditedPackages = try getActiveWorkspace().managedDependencies.hasEditedPackages()
```

Comment:
Worth a comment on why this is true, maybe with a bug reference if its not coming soon